src/lib/core/components/visualizations/implementations/BoxplotVisualization.tsx

```diff
@@ -257,13 +261,24 @@ function BoxplotViz(props: VisualizationProps) {
   );
 
   // outputEntity for OutputEntityTitle's outputEntity prop and outputEntityId at getRequestParams
-  const outputEntity = useFindOutputEntity(
+  let outputEntity = useFindOutputEntity(
```

Comment:
Would it be cleaner (but less readable!) with 

```
const outputEntity = computation.descriptor.type === 'abundance'
? entities.find(
      (e) =>
        e.id ===
        (computation.descriptor.configuration as ComputationConfiguration)
          ?.collectionVariable.entityId
    )
    : useFindOutputEntity(...
```

Reply:
Much cleaner! But it seems that react gets angry about violating the "rules of hooks" 😢 
```
React Hook "useFindOutputEntity" is called conditionally. React Hooks must be called in the exact same order in every component render  react-hooks/rules-of-hooks
```

Reply:
Oh, I see...

Reply:
What about
```
const outputEntity = useFindOutputEntity(...) ?? computation.descriptor.type === 'abundance'
? entities.find(
      (e) =>
        e.id ===
        (computation.descriptor.configuration as ComputationConfiguration)
          ?.collectionVariable.entityId
    ) : undefined;
```
?

Reply:
Works beautifully! Thanks @bobular ! I implemented your approach in the most recent commit